cif_pow.dic

```diff
@@ -11556,6 +11556,57 @@ save_PD_QPA_EXTERNAL_STD
     _name.object_id               PD_QPA_EXTERNAL_STD
     _category_key.name            '_pd_qpa_external_std.diffractogram_id'
 
+    _description_example.case
+;
+         data_ext_std_diffractogram
+            _pd_diffractogram.id         THE_REFERENCE
+            _pd_diffractogram.instr_id   labmachine
+
+            loop_
+            _pd_phase_mass.phase_id
+            _pd_phase_mass.absolute
+            _pd_phase_mass.absolute_su
+            NIST_ALUMINA_676A   99.02   1.11
+
+            _pd_qpa_external_std.k_factor           293.36
+            _pd_char.mass_atten_coef_mu_calc        3159
+            _pd_qpa_overall.method                  external_standard
+
+         data_diffractogram_block
+            _audit.schema                Custom
+            _pd_diffractogram.id         DIFFRACTOGRAM_2
+            _pd_diffractogram.instr_id   labmachine
+
+            loop_
+            _pd_phase_mass.phase_id
+            _pd_phase_mass.original
+            _pd_phase_mass.original_su
+            PHASE_1   42.81   0.56
+            PHASE_2   14.73   0.24
+
+            _pd_char.mass_atten_coef_mu_calc        6940
+            _pd_qpa_overall.method                  external_standard
+;
+    _description_example.detail
+;
+         In the first block, the K factor, or diffractometer constant, is
+         calculated from a diffraction pattern of a previously characterised
+         standard, collected under set conditions; see the
+         _pd_qpa_overall.method enumeration external_standard.
+
+         In the second block, a diffraction pattern containing two phases
```

Comment:
This explanation may need to be finessed if we switch from `instr_id` to `detector_id`.